platform/node/README.md

```diff
@@ -1,26 +1,22 @@
 # @maplibre/maplibre-gl-native
 
-[![NPM](https://nodei.co/npm/@maplibre/maplibre-gl-native.png)](https://npmjs.org/package/@maplibre/maplibre-gl-native)
+[![Version](https://img.shields.io/npm/v/@maplibre/maplibre-gl-native?style=flat)](https://www.npmjs.com/package/@maplibre/maplibre-gl-native)
 [![node-ci](https://github.com/maplibre/maplibre-gl-native/actions/workflows/node-ci.yml/badge.svg)](https://github.com/maplibre/maplibre-gl-native/actions/workflows/node-ci.yml)
 
 ## Installing
 
-#### :warn: This package isn't live yet.
-
-Requires a modern C++ runtime that supports C++14.
-
-By default, installs binaries. On these platforms no additional dependencies are needed.
+Binaries are available and downloaded during install for the following platforms:
 
 - 64 bit macOS or 64 bit Linux
```

Comment:
I would change this to something like

`MacOS 12 (amd64/arm64) or Ubuntu 20.04 (amd64/arm64)`

Reply:
Clarified. Also added a link to the debian instructions.